src/lib/blockies/makeBlockiesUrl.ts

```diff
@@ -0,0 +1,57 @@
+import type { BlockiesOptions } from './blockies'
+import { buildOpts, createImageData } from './blockies'
+
+const store: Record<string, string> = Object.create(null)
+
+const _btoa =
+  btoa || window?.btoa || ((str: string) => Buffer.from(str, 'binary').toString('base64'))
```

Comment:
Why do we polyfill this? This should be defined and if it's not, we always have it as a transient dep (but it should be defined in modern browsers so use the browser flavor)
<img width="478" alt="image" src="https://user-images.githubusercontent.com/17035424/234936789-0a1648a0-6bd6-4425-8986-416a2b391a00.png">